tests/topotests/ospf_gr_topo1/rt1/ospfd.conf

```diff
@@ -25,7 +25,6 @@ interface eth-rt2
 !
 router ospf
  router-id 1.1.1.1
- capability opaque
```

Comment:
Olivier is right. The `capability opaque` command will still be available, the only difference is that it will be enabled by default.

We could probably deprecate and remove this command at some point, as opaque LSAs are a fundamental part of OSPFv2 at this day and age. But I see no harm in keeping it around for longer.